srcDoc/release_notes.rst

```diff
@@ -48,6 +48,19 @@ releases:
 Release Notes Master
 --------------------
 
+What's New in Master
+~~~~~~~~~~~~~~~~~~~~
+
+Ability to use your own type in RTI Perftest
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*RTI Perftest* now supports the ability to use your own type in RTI Perftest.
+It is possible to measure the performance of your own type.
+
+The Custom Types feature allows the use of customized types instead of the one
+provided by RTI Perftest. It is designed in a way in which the number of changes
```

Comment:
It is designed in such a way that the number of changes ...

Reply:
So what does that mean (that changes in the code/configuration files is minimal)? Can you complete the sentence? e.g., "It is designed in such a way that the number of changes in the code and configuration files is minimal. So ..." (upgrading is easy? code generation is easy? or what? Or, if it's obvious to the user, no need to say anything).